packages/form-state-manager/src/utils/convert-value.ts

```diff
@@ -0,0 +1,23 @@
+import convertType from './convert-type';
+import { DataType } from '../types/data-types';
+
+const convertValue = (value: any, dataType: DataType): any => {
```

Comment:
This should only return a type specified in the `dataTypes` constant. Not any.

Reply:
With arrays and objects.. it's basically any. :D But I changed that.